### PR TITLE
New (hopefully not messy) PS2 update

### PR DESCRIPTION
**Pull request by rufotheone-fr**

### CODE REVIEW

File: clonelists/Sony - PlayStation 2.json [hunk 1]
```diff
@@ -159,6 +160,12 @@
 			["Aqua Aqua - Wetrix 2", 1],
 			["Aquaqua", 1]
 		],
+		"Ar tonelico: Melody of Elemia": [
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation 2.json:163]: Got a colon here. This should be `Ar tonelico - Melody of Elemia`

File: clonelists/Sony - PlayStation 2.json [hunk 2]
```diff
@@ -159,6 +160,12 @@
...
+			["Ar tonelico - Sekai no Owari de Utai Tsuzukeru Shoujo", 1]
+		],
+		"Ar tonelico II: Melody of Metafalica": [
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation 2.json:166]: Got a colon here. This should be `Ar tonelico II - Melody of Metafalica`.

**Review by unexpectedpanda — CHANGES_REQUESTED** [review]
Almost there :)

You've got some colons to fix and then we're done.

You can test how you're going by using the command line version of Retool and the `--errors` flag, which will report clone list mismatches as it goes along:

`retool.py <your dat> --errors`

It'll give you an output similar to the following image when it finds a problem:

errors

**Comment by rufotheone-fr** [comment]
Done!

**Comment by unexpectedpanda** [comment]
Nice! Thanks :)

**Pull request merged**